src/app/components/DeleteButton/DeleteButton.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import './DeleteButton.module.css';
+
+export default function deleteButton(): JSX.Element {
+  return <button>X</button>;
```

Comment:
You should use an aria attribute, if there is no 'real text' on the button. Also, you could thing about using an SVG in this case.

Reply:
maybe think about a button class where the text / svg inside the button is given to the component via a prop. in this way you dont have to write a new component for each button